src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/RiskLog.tsx

```diff
@@ -179,7 +178,6 @@ const RiskLog: React.FC<RiskLogProps> = ({ projectId, wbsNum, projLead, projMana
                     checked={risk.isResolved}
                     data-testid={`testCheckbox${idx}`}
                     onChange={() => handleCheck(risk)}
-                    disabled={isLoading}
```

Comment:
Was this line supposed to be deleted?

Reply:
Line 182 was not supposed to be deleted. My bad.

Reply:
all good just add it back